### ISSUE TITLE
Midi multi-track export

### ISSUE THREAD

**Issue by gloryofrobots** [hydrogen-music/hydrogen#692]
Hydrogen version   : 1.0.0.beta1   
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Currently, H2 exports all instruments into one midi track.

If H2 is used to create midi drum tracks for later work in some kind of DAW then multitrack export (one track per instrument) is more convenient, because there is often a need to EQ-se each drum separately. 

For my needs, I made a quick and dirty patch for the multitrack export [patch gist](https://gist.github.com/gloryofrobots/c309352e5f36153ede68e6338b089c11)
However, I think that more optimal is to have a midi export dialog where you can choose between single and multitrack export options
For implementation, I propose an addition of two new methods to SMFWriter class
```
saveMultiTrack( const QString& sFilename, Song *pSong )
saveSingleTrack( const QString& sFilename, Song *pSong )
```
and new UI class ExportMidiDialog similar to ExportSongDialog 
If this proposal gains approval from core developers I am willing to work on it

**Comment by trebmuh** [comment]
related to https://github.com/hydrogen-music/hydrogen/issues/580

**Comment by mauser** [comment]
Hi,

this has been implemented in #693, which has been merged to the master branch.
